src/utils/systemJsRendering.ts

```diff
@@ -1,12 +1,16 @@
 import Variable from '../ast/variables/Variable';
 
 export function getSystemExportStatement(exportedVariables: Variable[]): string {
-	if (exportedVariables.length === 1) {
+	if (exportedVariables.length === 1 && exportedVariables[0].exportName && exportedVariables[0].exportName.length === 1) {
 		return `exports('${exportedVariables[0].safeExportName ||
-			exportedVariables[0].exportName}', ${exportedVariables[0].getName()});`;
+			exportedVariables[0].exportName[0]}', ${exportedVariables[0].getName()})`;
 	} else {
-		return `exports({${exportedVariables
-			.map(variable => `${variable.safeExportName || variable.exportName}: ${variable.getName()}`)
-			.join(', ')}});`;
+		return `exports({ ${exportedVariables
+			.map(variable =>
+				// TODO: do we always have export names at this point?
+				variable.exportName!.map(exportName => `${exportName}: ${variable.getName()}`)
+				.join(', ')
+			)
+			.join(', ')} })`;
```

Comment:
There previously were not whitespace placeholders here, let me know if this should respect `compact` better than it currently does.